src/main/java/uk/gov/justice/hmpps/prison/service/ImageService.java

```diff
@@ -68,8 +68,7 @@ public Page<OffenderNumber> getOffendersWithImagesCapturedAfter(final LocalDateT
             .map(offender -> new OffenderNumber(offender.getNomsId()));
     }
 
-    @PreAuthorize("hasRole('IMAGE_UPLOAD')")
-    @HasWriteScope
+    @PreAuthorize("hasRole('IMAGE_UPLOAD') and hasAuthority('SCOPE_write')")
```

Comment:
spring boot 3 got upset here as there are multiple `PreAuthorize` annotations and there is now a check to ensure that only one exists.  I've therefore had to combine them both into a single one